web/templates/web/faq.html

```diff
@@ -603,9 +631,8 @@ <h2 class="accordion-header" id="heading-23">
-                <p>We decided it would be nice for there to be one place online where parents and researchers could go to connect with each other to support research into child development!</p>
+                <p>If you are a parent or guardian who has participated with your child in a study on this website, or a university-based researcher who has posted a study on this website, then we consider you a member of the Parent Researcher Collaborative!</p>
+                <p>If you are asking who runs this website, the answer is that we are a group of scientists who decided it would be nice for there to be one place online where families and researchers could go to connect with each other to support research into child development! The new Children Helping Science website is a collaboration between two previous platforms, Lookit and (an earlier version of) Children Helping Science. Lookit was founded by <a href = "https://kimberscott.github.io/">Kim Scott</a> and <a href = "http://eccl.mit.edu/">Laura Schulz</a> at MIT, and is now lead by Executive Director <a href = "http://www.melissaklinestruhl.com/">Melissa Kline Struhl</a>.  The original Children Helping Science website was created by <a href = "https://projects.iq.harvard.edu/ccdlab">Elizabeth Bonawitz</a> at Harvard, <a href = "http://sll.stanford.edu/">Hyowon Gweon</a> at Stanford, <a href = "https://compdevlab.yale.edu/">Julian Jara-Ettinger</a> at Yale, <a href = "https://www.utdallas.edu/thinklab/">Candice Mills</a> at UT Dallas, <a href = "http://eccl.mit.edu/">Laura Schulz</a> at MIT, and <a href = "https://www.minerva.kgi.edu/people/mark-sheskin-phd/">Mark Sheskin</a> at Minerva University. Mark did the web development (based on an initial design by Junyi Chu, a student in Laura's lab) and the logo was designed by Natalia Vélez, a student in Hyowon's lab. </p>
```

Comment:
I think the [comment](https://github.com/lookit/lookit-api/issues/1179#issuecomment-1529951452) about this section from the FAQ issue also says to remove the last sentence in this paragraph.

Reply:
The comment on the issue has two paragraphs.  The remaining text was removed.  Here's the comment from the issue and the FAQ in the local environment.  

Issue comment:
<img width="824" alt="Screenshot 2023-05-11 at 9 58 33 AM" src="https://github.com/lookit/lookit-api/assets/44074998/d8aabe54-3c73-4e91-9e04-cd822756d395">

FAQ:
<img width="1300" alt="Screenshot 2023-05-11 at 9 59 13 AM" src="https://github.com/lookit/lookit-api/assets/44074998/44e9bc07-7e52-4550-b08c-b35832c6edbd">